index.html

```diff
@@ -401,6 +401,12 @@ <h3>eddsa-2022</h3>
 section also include the verification of such a data integrity proof.
         </p>
 
+        <p class="advisement">
+When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is utilized,
+implementations MUST detect <a data-cite="RDF-CANON#dataset-poisoning">
```

Comment:
```suggestion
implementations will detect <a data-cite="RDF-CANON#dataset-poisoning">
```

Does this work for you @dlongley, @iherman, @pchampin, and @OR13?

Reply:
This works for me, if merged, i will approve.